wandb/sdk/launch/inputs/internal.py

```diff
@@ -62,11 +62,13 @@ def __init__(
         self,
         include: Optional[List[str]] = None,
         exclude: Optional[List[str]] = None,
+        input_schema: Optional[dict] = None,
```

Comment:
could we call this `schema` instead of `input_schema` actually? I think it's a little bit cleaner, and this is a user facing function